fabric-fluids-v1/logs/latest.log

```diff
@@ -0,0 +1 @@
+[21:27:54] [main/ERROR]: Unable to bootstrap registry 'minecraft:fluid'
```

Comment:
I assume this file should be in .gitignore?

Reply:
Yep. We eclipse users really need the .gitignore PR merged. I have to fuss with it every time I reset my branch and then hope I don’t check it in.